<launch>
    <arg name="map_server" default = "1" />
    <arg name="cars_file" default = "2cars.yaml" />
    <arg name="tasks_file" default = "2cars2tasks3x5.yaml" />
    <rosparam file="$(find mushr_coordination)/config/$(arg cars_file)" command="load" ns="init_clcbs" />
    <rosparam file="$(find mushr_coordination)/benchmark/$(arg tasks_file)" command="load" ns="init_clcbs" />

    <!-- Launch  map server-->
    <group if="$(arg map_server)">
        <include file="$(find mushr_base)/launch/includes/map_server.launch" />
    </group>

    <node pkg="mushr_coordination" type="init_planner.py" name="init_planner" output="screen" />
</launch>
